plugins/reach/ReachPlugin.php

```diff
@@ -331,6 +333,26 @@ public static function buildDataOnTask(VendorCatalogItem $catalogItem, $partnerI
 		return $data;
 	}
 
+    /**
+     * @inheritDoc
+     */
+    public static function getTranslations($locale)
+    {
+        $array = array();
+
+        $langFilePath = __DIR__ . "/config/lang/$locale.php";
+        if(!file_exists($langFilePath))
+        {
+            $default = 'en';
```

Comment:
Use const from KalturaLanguageCode

Reply:
@MosheMaorKaltura  at this point in the flow I'm not certain that the const class it loaded. I'll put a pin in this comment and test the change. If it's possible to use the const, I'll add this fix later.